blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ScheduleContext.java

```diff
@@ -481,7 +480,12 @@ public long read() {
         return results;
     }
 
-    private void registerMBean() {
+    private boolean registered = false;
```

Comment:
@ChandraAddala what about `mbeanHasBeenRegistered`? I think that would clarify *what* it is that is being registered. Or what about `isMbeanRegistered`?